services/database/relational/tests/ccc-rdms-c8.feature

```diff
@@ -0,0 +1,18 @@
+@CCC.RDMS.C8.TR01
+Feature: Data encryption in transit
+
+"""
+This reature ensures that end-to-end encryption of data in transit is leveraged and enforced
+"""
+
+@CCC.RDMS.C8.TR01.T01
+Scenario: Verify that databases are enforcing encrypted connections
+   Given an application attempting to connect to a database and the database is configured with some form of "require secure transport"
+   When the connection attempt is made without using encryption
+   Then the connection should be refused
+
+@CCC.RDMS.C8.TR01.T02
+Scenario: Ensure ability to audit for any users who have connected using an insecure protocol
+   Given a user connection has been established to the database
+   When an admin establishes an admin connection to a database server and runs from mysql.user where ssl_type=''
```

Comment:
For our controls, we try to keep them as abstract as possible. we should remove the mysql.user where ssl_type='' from this statement.

Reply:
Makes sense!  If I'm otherwise on track, I'll push a few more over in addition to threats/controls.

Reply:
Also remember to keep an eye on the new [ID format](https://github.com/finos/common-cloud-controls/blob/main/docs/governance/community-guidelines/content-standards-and-practices/identifiers.md). Most notably the Ts... threats are now `TH00`, Test requirements are `TR00` and Tests are `TE00`